tasks/cycles/cycles.js

```diff
@@ -13,7 +13,17 @@
 function calculateDiscount(redemption) {
   let discount;
 
-  //PLACE YOUR CODE HERE
+  if (redemption <= 350){
+    console.log('0%');
```

Comment:
Do not use console.log() here. This task requires running of a test: npm run test:cycles, so you need to do the folowing:
if (redemption <=350) {
   discount = 0;
}
....
and etc